### ISSUE TITLE
Add DSA code

### ISSUE THREAD

**Issue by Pradeepsingh61** [Pradeepsingh61/DSA_Code#1]
## INSTRUCTIONS-

- Fork this Repository using the button at the top on the right corner.
- Clone your forked repository to your PC ( git clone "url from clone option.)
- Create a new branch for your modifications (ie. `git branch new-user` and check it out  `git checkout new-user` or simply do `git checkout -b new-user`)
- Add your profile image in `static/images/` ( use drag and drop option or upload by commands.)
- Add your profile data in `content/participant/`
- Add your files (`git add -A`), commit (`git commit -m "added myself"`) and push (`git push origin new-user`)
- Create a pull request
- Star this repository

# How To Make Your First Pull Request

```

<div align="center">
<h2> OR </h2>
</div>

## 1. Create / Upload Your Code / Algorithms

Create/Upload your code in the folder following the convention `Add Code Here`
- Choose an extract language folder `drop your code`
- Below is `an example` to add file properly
- You can follow up `any languages`

/PYTHON/<YOUR-FILERNAME>.py

/C++/<YOUR-FILERNAME>.cpp

You can follow any pathway a `code` or `profile information`

## 2. Wait for Pull Request to merge

## 3. Celebrate - you've done your first pull request!!

'''
Always make more than 4 pull requests.
Let's say you have made only 4 pull requests to different projects,
but one project is excluded from hackoctoberfest event then your pull request will not be counted and 
then you have the remaining 3 valid pull requests if these projects are not excluded.
If you fail to make 4 pull requests then you can't get swags or t-shirts.
I will recommend you make a pull request to your own repo which is very very safe for you.
Keep in mind that the repo has hacktoberfest topic.
'''
```

## + Follow Me : } Quick Approval of Pull Request

```py
'''
To get approval of the pull request much quicker and faster (`Follow Me`)🚀
'''

**Comment by Karanjot786** [comment]
Assignees this issue

**Comment by vidhvath28** [comment]
I would like to contribute to this issue, but kindly add the hacktoberfest or hacktoberfest-accepted this would really help us out

**Comment by Diwansu** [comment]
Assign this issue to me.

**Comment by Vrajjangid5** [comment]
I also want to contribute this issue

**Comment by shivigoyal4321** [comment]
please assign this to me, i would like to contribute! @Pradeepsingh61

**Comment by Pradeepsingh61** [comment]
Hi @shivigoyal4321, just open a new PR. I will accept all your PRs

**Comment by Ruksina01** [comment]
Hey @Pradeepsingh61 ,
I am interested in this issue. Could you please assign it to me? Thank you!

**Comment by kaustubhudavant-hash** [comment]
I want to contribute in this issue for hacktoberbest 2025.please assign it to me.

**Comment by kaustubhudavant-hash** [comment]
Hi @Pradeepsingh61 please assign to me,I would like to contribute for hacktoberbest 2025

**Comment by Karanjot786** [comment]
Hi @kaustubhudavant-hash and @JitinSaxenaa, welcome to our DSA code repo! Please feel free to open PRs and issues following the guidelines here: [CONTRIBUTING.md](https://github.com/Pradeepsingh61/DSA_Code/blob/main/CONTRIBUTING.md). No need to assign this issue.

**Comment by JitinSaxenaa** [comment]
@Karanjot786 Hi Sir, I have implemented all the required changes. However, the CI compilation fails due to some missing files that are unrelated to my contribution. These files are not needed for the successful compilation and execution of my Sparse Matrix code. Please review the PR.

**Comment by Karanjot786** [comment]
@JitinSaxenaa yes, I see you need to fix these issues in the first two PRs #184 and #185

**Comment by JitinSaxenaa** [comment]
@Karanjot786 I have resolved all the issues,please review.

**Comment by SrinjoyeeDey** [comment]
👋 Hello @Pradeepsingh61 ,

I have raised #193  issue and I would love to work on it. It would be really motivating if you could kindly assign it to me and also add the Hacktoberfest-accepted label so that my contribution can be counted.

I assure you I will raise a PR very soon once you confirm the assignment. 🙏
Thank you for considering! ✨

**Comment by SrinjoyeeDey** [comment]
Hi @Pradeepsingh61 , I noticed that #193 issue already has an assignee just few seconds after I created it. I’m curious—does the repo have a default assignment workflow, or was it manually assigned? I want to clarify it before raising any other issues. Thank You .

**Comment by Karanjot786** [comment]
Hi @SrinjoyeeDey, welcome to our DSA code repo! I am the reviewer of this repo, and it's the default assignment to me, and no need to worry about and I have assigned it to you now, so you can open the PR

**Comment by AKG1301** [comment]
Please assign this to me, I would like to contribute! @Pradeepsingh61

**Comment by JitinSaxenaa** [comment]
> Hi [@JitinSaxenaa](https://github.com/JitinSaxenaa), I am closing your all the PRs and now open the only one PR at a time and the issue comes because your all the PRs is overlapping and it is easy to us to review and merge it

Means have you deleted all of my pr's? in my hacktober dashboard its showing 8 PR's in review since yesterday. will my old prs cound or have you deleted it? what should i do now?

@Pradeepsingh61
